libs/clients/syslumenn/src/lib/syslumennClient.service.ts

```diff
@@ -215,6 +223,41 @@ export class SyslumennService {
     return response.map(mapDistrictCommissionersAgenciesResponse)
   }
 
+  async getAsset(
+    assetId: string,
+    assetType: AssetType,
+    assetMapper: (res: VedbandayfirlitReguverkiSvarSkeyti) => AssetName,
+  ): Promise<Array<AssetName>> {
+    const { id, api } = await this.createApi()
+    const response = await api
+      .vedbokavottordRegluverkiPost({
+        skilabod: {
+          audkenni: id,
+          fastanumer: assetId,
+          tegundAndlags: assetType as number,
+        },
+      })
+      .catch((e) => {
+        if ((e as { status: number })?.status === 404) {
+          return []
+        }
+        throw e
+      })
+
+    console.log({
```

Comment:
Perhaps remove the console.log. Unless this is intentional and you still want this to be logged for some reason :P

Reply:
Ah yes, I thought I had gone through these but here we are. Thank you :D